



### **Workshop Details:**
This **hands-on** workshop spans **5 consecutive days** with 4.5 hours of synchronous teaching time and 2 hours of homework per day. The workshop is designed for beginners without any programming experience. ***All sessions will be held online over Zoom.***

**Morning sessions**: 9:30 AM - 12 PM EST

**Afternoon sessions**: 1:00 PM - 3:00 PM EST

---

### **Description:**
This workshop begins with an introduction to using R and RStudio. Participants will learn the basics of R syntax and how to utlilize functions and packages in R for reproducible data wrangling and visualizations. Next, we will begin data analysis in R using single cell RNA-seq count data. Participants will use Seurat and associated tools to perform analysis of single-cell expression data, including data filtering, QC, integration, clustering, and marker identification analysis. Finally, the workshop will demonstrate pseudobulk and other approaches for identifying differential expression of genes across samples groups.

---

### **Who should attend?**

Any interested individuals who are keen in obtaining a foundational understanding of the workflow for analyzing single cell RNA-seq data in R. 

***We encourage academic and industry researchers who are working on scRNA-seq data or have plans to embark on single cell experiments in the near future to apply to this workshop.***

---

### **Registration:**
To register for the course please click on the link below. If you are one of the *first 25 registrants*, you will receive an email within one week with a link to pay the (non-refundable & non-transferable) registration fee.

<!--
<div style="text-align:center">
	 <a><button name="button" style = "color: red" >Registration is now closed!</button></a>
</div>

-->

<div style="text-align:center">
	 <a><button name="button" style = "color: blue" onclick="location.href='https://harvard.az1.qualtrics.com/jfe/form/SV_9MFlOEFb3W5AfBA'">Click here to Register!</button></a>
</div>


<!-- This content will not appear in the rendered Markdown -->


---

### **Cost:**

There is a _non-refundable_ and _non-transferable_ registration fee for this workshop. The registration fee options are outlined below.

***Early Bird Pricing available until June 6th!***

| Rate Category | Early Bird  | Regular Rate| 
|:---------------|:-------------:|:-------------:|
| Harvard Academic  | $860 USD | $960 USD |
| External Academic | $1125 USD | $1250 USD |
| Industry | $1575 USD | $1750 USD |


Due to limited space the workshop can accommodate **maximum of 25 participants**. Seats are assigned on a **first come, first serve** basis.

 
---

### **Workshop Outline:**

### **Day 1:**
* **R syntax**: Understanding the different 'parts of speech' in R; introducing variables and functions, demonstrating how functions work, and modifying arguments for specific use cases.
* **Data structures in R**: Getting a handle on the classes of data structures and the types of data used by R. Reading data into R and using functions to inspect it.
  
### **Day 2:**
* **Data inspection and wrangling**: Using indices and various functions to subset and create datasets, including the tidyverse suite of packages.
* **Visualizing data**: Visualizing data using plotting functions from the external package ggplot2.
* **Exporting data and graphics**: Generating new data tables and plots for use outside of the R environment.


### **Day 3:**
* **Experimental considerations for scRNA-seq**: A practical guide on setting up a successful single cell experiment and detailed information on different platforms.
* **Quality control**: Understanding cellranger output and a review of the web summary HTML report. Exploring additional quality control metrics using the Seurat object and various data visualizations in R. 
* **Normalization**: Exploring unwanted variation in the data and applying various normalizations to the data.

### **Day 4:**
* **Integration**: Approaches for combining samples within a dataset, and across datasets.
* **Clustering cells**: Identifying groups of similar cells based on expression data and and visualization of clusters on a UMAP.
* **Clustering QC**: Evaluate whether clustering artifacts are present. Assess known cell type markers to derive the identity of the different cell types present amongst clusters.

### **Day 5:**
* **Pseudobulk**: Aggregating counts by celltype in preparation for differential gene expression analysis.
* **Differential expression analysis**: Using DESeq2 to identify differentially expressed genes between sample groups for a given celltype.
* **Data Visualization**: Comparing the results of the pseudobulk approach to a `FindMarkers()` analysis using data visualizations.


---

### **Questions?**

Please email us at hbctraining@hsph.harvard.edu with any questions.
